AnnoyancesFilter/Cookies/sections/cookies_specific.txt

```diff
@@ -277,6 +277,7 @@ foto-erhardt.com,foto-erhardt.de,seriouseats.com,fotogena.de,gqmagazine.fr,rossm
 !
 ! NOTE: Regular rules
 !
+rapfame.app##.jss341
```

Comment:
Better to avoid such selectors, but you can add them if there is no other choice.
This rule works fine on my end. I will change it.
```adblock
rapfame.app###header > div[class^="MuiCollapse-container-"][style^="min-height:"]
```